pkg/deploytest/deployment.go

```diff
@@ -228,7 +228,7 @@ func localGrpcTransport(nodeIds []t.NodeID, ownId t.NodeID) *grpctransport.GrpcT
 	// Each test replica is on the local machine - 127.0.0.1
 	membership := make(map[t.NodeID]string, len(nodeIds))
 	for _, id := range nodeIds {
-		p, err := strconv.Atoi(string(id))
+		p, err := strconv.Atoi(id.Pb())
```

Comment:
Could you extend the comment documenting the function (line 223-224), and potentially other places if appropriate, that in the test deployment the node ID strings must be parseable to decimal numbers?

Reply:
ok